docs/configuring-metabase/custom-maps.md

```diff
@@ -19,27 +19,40 @@ To find the map settings:
 
 ## Map tile server
 
-By default, Metabase uses the OpenStreetMaps tile server. If your organization requires a different
-look or level of detail in your map visualizations, you can change the map tile server by adding the map tile server path to the first field on the page. The path must either be a URL that starts with `http://` or `https://` or a relative path to a local file in the JVM's classpath.
+By default, Metabase uses the [OpenStreetMaps](https://www.openstreetmap.org) tile server for pin and grid maps. If your organization requires a different
+look or level of detail in your map visualizations, you can change the map tile server by adding the map tile server path to the first field on the page.
+
+The path must either be a URL that starts with `http://` or `https://` or a relative path to a local file in the JVM's classpath.
+
+Currently, Metabase uses a single tile server per instance. You can't specify different tiles for different maps.
 
 ## Custom region maps
 
-If you need a map that focuses on a specific region, you may want to upload a custom GeoJSON map.
+Metabase comes with two built-in [region maps](../questions/visualizations/map.md#region-maps): world map with countries and United States map with states. If you need a map of other regions - like EU countries or NYC neighborhoods - you can upload a GeoJSON file containing region information.
+
+If you need a GeoJSON file for specific custom regions, you can often find it just by searching online for "[Your region] + geojson", like "NYC neighborhoods GeoJSON", because community members and government organization have already developed map files for many common regions. You can also create your own GeoJSON with a tool like [MapShaper](https://mapshaper.org/) or [GeoJSON.io](http://geojson.io/).
```

Comment:
This is added because we frequently get asked "but where would i get a geojson". Idk what's a nice way of saying lmgtfy